vnext/Microsoft.ReactNative.Cxx/JSValueTreeReader.h

```diff
@@ -1,6 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+///////////////////////////////////////////////////////////////////////////////
+//                              IMPORTANT
```

Comment:
>IMPORTANT [](start = 32, length = 9)

It would be nice if we do not put such non-API specific warning in each file. Please created a README.md file in the project instead.
We can help a one line instead in each file asking to read the README.md in case if any changes are done.

Reply:
Updated with `README.md`